.checkov.baseline

```diff
@@ -97,7 +97,14 @@
                     "check_ids": [
                         "CKV_AWS_337"
                     ]
+                },
+                {
+                    "resource": "module.extensions-staging.module.elasticache-redis.aws_ssm_parameter.endpoint_ssl",
+                    "check_ids": [
+                        "CKV_AWS_337"
+                    ]
```

Comment:
Why have we added this to the baseline instead of addressing the problem it raised?

Reply:
The same error appears in the original module.extensions-staging.module.elasticache-redis.aws_ssm_parameter.endpoint resource and that's how it was handled there.

I thought addressing the contents of the checkov.baseline was a separate ticket?

Reply:
The idea of the baseline file is too allow us to start using Checkov without getting lost in having to fix all the existing problems. So we can adopt it without disruption and use it to guard against adding any further misconfigurations. It is not intended to be used to skip any further checks.

If there is a valid reason to skip a check, like we have decided to do something in a way Checkov doesn't like for a deliberate reason, the checks can be skipped in the Terraform manifests with a comment explaining the reason.

Reply:
I get you now. I'll add the kms key for ssm string encryption on both redis endpoint types and remove them from the checkov baseline file.

Reply:
Added kms key policy